examples/blink/CMakeLists.txt

```diff
@@ -27,12 +27,17 @@ emil_build_for(hal_st.blink_nucleo64
     TARGET_MCU stm32g431
     TARGET_MCU stm32g474
     TARGET_MCU stm32wb55
+    TARGET_MCU stm32wba52
     PREREQUISITE_BOOL HALST_BUILD_EXAMPLES
 )
 
+
 target_compile_definitions(hal_st.blink_nucleo64 PUBLIC
-    $<$<NOT:$<STREQUAL:${TARGET_MCU},stm32wb55>>:NUCLEO=Nucleo64Ui>
+    $<JOIN:stm32g4x,stm32g431>
+    $<JOIN:stm32g4x,stm32g474>
+    $<$<STREQUAL:${TARGET_MCU},stm32wba52>:NUCLEO=Nucleo64WBAUi>
     $<$<STREQUAL:${TARGET_MCU},stm32wb55>:NUCLEO=Nucleo64WBUi>
+    $<$<IN_LIST:${TARGET_MCU},stm32g4x>:NUCLEO=Nucleo64Ui>
```

Comment:
Ah, you're trying to construct a list stm32g4x. It doesn't work like that. $<JOIN:...> or any other kind of generator expression does not build up state, but returns a string.
I expect that $<IN_LIST:${TARGET_MCU},"stm32g431,stm32g474"> does what you want

Reply:
Thanks for the suggestion, done.